en/docs/administer/monitoring-logs.md

```diff
@@ -104,33 +106,38 @@ appender.AUDIT_LOGFILE.policies.type = Policies
 appender.AUDIT_LOGFILE.policies.time.type = TimeBasedTriggeringPolicy
 appender.AUDIT_LOGFILE.policies.time.interval = 1
 appender.AUDIT_LOGFILE.policies.time.modulate = true
-appender.AUDIT_LOGFILE.policies.size.type = SizeBasedTriggeringPolicy
-appender.AUDIT_LOGFILE.policies.size.size=10MB
 appender.AUDIT_LOGFILE.strategy.type = DefaultRolloverStrategy
 appender.AUDIT_LOGFILE.strategy.max = 20
 appender.AUDIT_LOGFILE.filter.threshold.type = ThresholdFilter
 appender.AUDIT_LOGFILE.filter.threshold.level = INFO
 ```
 
+The log levels that
+can be configured are [listed below](#MonitoringLogs-log4j-levels).
+
 #### Configuring HTTP access logs
 
 See [HTTP Access Logging](../../administer/http-access-logging) for instructions on
 how to configure and use HTTP access logs.
 
+#### Configuring product observability
+
+WSO2 Identity Server supports logging capabilities for tracking down latencies due to database calls.
+See [Working with Product Observability](../../administer/working-with-product-observability) for instructions on
+how to configure and use this capability.
+
 #### Setting the Log4j log level
 
 The log level can be set specifically for each appender in the
-`         log4j.properties        ` file by setting the threshold value.
+`         log4j2.properties        ` file by setting the threshold value.
 If a log level is not specifically given for an appender as explained
 below, the root log level (INFO) will apply to all appenders by default.
 
+Following is how the log level is set to `DEBUG` for the `CARBON_LOGFILE` appender ([Carbon log](#configuring-carbon-logs)): 
 
 !!! example  
-
-    Following is how the log level is set to `DEBUG` for the `CARBON_LOGFILE` appender ([Carbon log](#configuring-carbon-logs)): 
-    ``` java
+   
     appender.CARBON_LOGFILE.filter.threshold.level = DEBUG
```

Comment:
Shall we put this inside a code block? 

appender.CARBON_LOGFILE.filter.threshold.level = DEBUG

To put it inside a code block, add ``` to the beginning and end

Reply:
done